packages/core/src/notification/selector/types/cdevents/CDEventsNotificationType.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import type { INotificationTypeCustomConfig } from '../../../../domain';
+import { FormikFormField, TextInput, Validators } from '../../../../presentation';
+
+export class CDEventsNotificationType extends React.Component<INotificationTypeCustomConfig> {
+  public render() {
+    const { fieldName } = this.props;
+    return (
+      <>
+        <FormikFormField
+          label="Events Broker URL"
+          name={fieldName ? `${fieldName}.address` : 'address'}
+          validate={Validators.skipIfSpel(Validators.urlValue('Please enter a valid URL'))}
+          input={(props) => (
+            <TextInput
+              inputClassName={'form-control input-sm'}
+              {...props}
+              placeholder="URL starts with https://events-broker-address/default/events-broker/"
```

Comment:
Placeholder text is best when it is an actual possible value, not a description (e.g. `placeholder="https://..."`).  

The screenshot in the PR already has it like that, but this is different here in the code.  

More descriptive help can be added via a `help` field on the `FormikFormField` component, e.g.:

```
<FormikFormField
  ...
  help={<HelpField id="..." />}
  ...
  input={...}
/>
```

We may also need this placeholder value configurable from `settings.js`, as it would need to match the custom validation.

Reply:
I followed the same format how it is used today for MS Teams or GoogleChat notifications, not sure If we change for this alone, will it be a different user experience?

Configurable from settings.js, not sure If I understood this correctly is that keeping a key:value pair in settings.js something like [this](https://github.com/spinnaker/deck/blob/master/packages/app/src/settings.js#L144), but this will actually keep the default value right, rather this value should be just an example URL

Reply:
I don't mind that much either way on the content of the placeholder text.  However, if the URL validation is configurable, the placeholder value should not conflict with it.  

I would probably suggest removing it in this case, as the path on the event broker is not in the CDEvents spec, and so the `/default/events-broker/` path is probably more confusing than helpful.

Reply:
Ok, changing the placeholder to a message "Enter an events message broker URL" and make configurable through `settings`